### PR TITLE
Intermediate results for network inspect scan

### LINKED ISSUE #647: Improve network scan intermediate status
# Opening a new issue
___

## Specify type:
- Enhancement

### Bug severity (if applicable):
- High

___

## Description:
Currently, a network scan will only log info when the host is complete.  This could be slow depending on the concurrency setting (may need 50 hosts to complete before you see logs).  This work will add a log entry when a role is started for a host.
___

### PR DESCRIPTION
**Pull request by kholdaway**
Closes #647

### CODE REVIEW

File: quipucords/scanner/network/inspect.py [hunk 1]
```diff
@@ -188,7 +188,14 @@ def inspect_scan(self, connected, roles=DEFAULT_ROLES,
         inventory_file = write_inventory(inventory)
 
         error_msg = ''
+        group_index = 0
+        log_message = 'START PROCESSING GROUPS of size %d' % forks
+        self.scan_task.log_message(log_message)
         for group_name in group_names:
```
Comment (chambridge) [inline quipucords/scanner/network/inspect.py]: suggestion for more pythonic:
```
for idx, val in enumerate(ints):
    print(idx, val)
```

File: roles/host_done/tasks/main.yml [hunk 2]
```diff
@@ -1,5 +1,9 @@
 ---
 
+- name: internal_host_started_processing_role
+  set_fact:
+    internal_host_started_processing_role: "host_done"
+
 - name: extract virt-what error code
```
Comment (chambridge) [inline roles/host_done/tasks/main.yml]: Seems like a bad task name that came in.

File: quipucords/scanner/network/inspect.py (hunk 1 above)
Comment (kholdaway) [reply quipucords/scanner/network/inspect.py]: Great suggestion.

File: roles/host_done/tasks/main.yml (hunk 2 above)
Comment (kholdaway) [reply roles/host_done/tasks/main.yml]: Fixed.

**Pull request merged**